src/engine/ruleset/integrations/windows/decoders/windows-event.yml

```diff
@@ -28,6 +28,8 @@ normalize:
 
       - host.name: $windows.System.Computer.#text
 
+      - message: kvdb_get(windows_id_to_message, $event.id)
+
       - _win_log_level:
           "0": "Audit"
```

Comment:
Should double quotes be replaced with single quotes, right? And removed double quotes in values.

```
- _win_log_level:
          '0': Audit
          '1': Critical
```

Reply:
done